src/optionalActions.ts

```diff
@@ -2,13 +2,6 @@ import {Content, Repository} from './main'
 import {Octokit} from 'octokit'
 import * as core from '@actions/core'
 
-export const removeToken = (content: Content | null) => {
-  if (content && content.downloadUrl) {
-    content.downloadUrl = content.downloadUrl.replace(/\?(.*)/, '')
-  }
-  return content
-}
-
```

Comment:
Is there no token in the download url anymore?

Reply:
That is managed here : https://github.com/vineeth-asml/load-available-actions/blob/e4519757b0a1c47a40f737640dff86e1f8c96c51/src/main.ts#L205